packages/toolpad-app/src/components/EditableText.tsx

```diff
@@ -1,39 +1,94 @@
 import * as React from 'react';
-import { Skeleton, TextField, Typography, TypographyVariant, Tooltip } from '@mui/material';
+import {
+  TextField,
+  Theme,
+  TypographyVariant,
+  SxProps,
+  InputBaseComponentProps,
+} from '@mui/material';
 
 interface EditableTextProps {
   defaultValue?: string;
-  loading: boolean;
-  editing: boolean;
-  isError: boolean;
+  editing?: boolean;
+  isError?: boolean;
   errorText?: string;
-  onKeyUp: (event: React.KeyboardEvent<HTMLInputElement>) => void;
-  onBlur: (event: React.FocusEvent<HTMLInputElement>) => void;
+  onKeyUp?: (event: React.KeyboardEvent<HTMLInputElement>) => void;
+  onBlur?: (event: React.FocusEvent<HTMLInputElement>) => void;
   variant?: TypographyVariant;
   size?: 'small' | 'medium';
+  sx?: SxProps;
+  helperText?: React.ReactNode;
+  disabled?: boolean;
+  inputProps?: InputBaseComponentProps;
```

Comment:
I don't see this prop used anywhere. If there's no use for it, I would remove it. No need to leak abstraction if we don't have to.